self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryContracts.java

```diff
@@ -27,17 +27,20 @@
 import com.selfxdsd.core.contracts.ContributorContracts;
 import com.selfxdsd.core.contracts.ProjectContracts;
 
+import java.math.BigDecimal;
 import java.util.*;
 import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
 /**
  * In-Memory Contracts for testing purposes.
  *
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
+ * @todo #74:30min When Storage API has contributors available, update
+ * InMemoryContracts#getContributorById() to use storage. Then update
+ * InMemoryContractsTestCase tests to reflect that.
```

Comment:
@criske line 2...n of a Puzzle should always start with a space (they should be aligned after the ``@``), otherwise 0pdd will complain.